Source/Core/Common/IniFile.h

```diff
@@ -111,7 +111,7 @@ class IniFile
 	// Returns true if key exists in section
 	bool Exists(const std::string& sectionName, const std::string& key) const;
 
-	template<typename T> bool GetIfExists(const std::string& sectionName, const std::string& key, T value)
+	template<typename T> bool GetIfExists(const std::string& sectionName, const std::string& key, T* value)
```

Comment:
This was changed to a pointer, since a pointer is always expected here and it is necessary for properly overloading the function below.